app/models/SicAndCompliance.scala

```diff
@@ -42,25 +39,17 @@ object SicAndCompliance {
     val labourComp = (json \ "labourCompliance").validateOpt[JsObject].get
     val numOfWorkers = labourComp.map(a => (a \ "numberOfWorkers").as[Int])
     val workers = numOfWorkers.flatMap(num => if (num == 0) None else Some(Workers(num)))
-    val temporaryContracts = workers.flatMap { _ =>
-      (json \ "labourCompliance" \ "temporaryContracts").validateOpt[Boolean].get.map { b =>
-        if (b) TemporaryContracts(TEMP_CONTRACTS_YES) else TemporaryContracts(TEMP_CONTRACTS_NO)
-      }
-    }
-    val skilledWorkers = workers.flatMap { a =>
-      (json \ "labourCompliance" \ "skilledWorkers").validateOpt[Boolean].get.map { b =>
-        if (b) SkilledWorkers(SKILLED_WORKERS_YES) else SkilledWorkers(SKILLED_WORKERS_NO)
-      }
-    }
+    val temporaryContracts = (json \ "labourCompliance" \ "temporaryContracts").validateOpt[Boolean]
+      .getOrElse(None)
+      .map(IntermediarySupply.apply)
```

Comment:
Backend is currently using old labels - once it has been updated we can do a quick PR to update these